data/cube-hex-ro.mesh

```diff
@@ -0,0 +1,86 @@
+MFEM mesh v1.0
+
```

Comment:
For early testing I prefer to use either the inline mesh generator or the Mesh(...) constructor that generates simple cuboidal meshes.  It's really easy to mess up the numbering when doing these by hand.